apps/enc.c

```diff
@@ -51,7 +51,7 @@ typedef enum OPTION_choice {
 const OPTIONS enc_options[] = {
     {"help", OPT_HELP, '-', "Display this summary"},
 #if !OPENSSL_API_3
-    {"ciphers", OPT_LIST, '-', "Alias of -list"},
+    {"ciphers", OPT_LIST, '-', "alias for -list"},
```

Comment:
Oh, but it was not my intention that you change it to lower case. The other help texts also start with a capital letter.

Reply:
Thank you!  Resolved in my latest commit.